core/server/data/migrations/versions/3.22/02-settings-key-renames.js

```diff
@@ -0,0 +1,81 @@
+const logging = require('../../../../../shared/logging');
+
+const renameMapping = [{
+    from: 'default_locale',
+    to: 'lang'
+}, {
+    from: 'active_timezone',
+    to: 'timezone'
+}, {
+    from: 'ghost_head',
+    to: 'codeinjection_head'
+}, {
+    from: 'ghost_foot',
+    to: 'codeinjection_foot'
+}];
+
+module.exports = {
+    config: {
+        transaction: true
+    },
+
+    async up(options) {
+        await Promise.map(renameMapping, async (renameMap) => {
+            logging.info(`Renaming ${renameMap.from} to ${renameMap.to}`);
+
+            return await options
+                .transacting('settings')
+                .where('key', renameMap.from)
+                .update({
+                    key: renameMap.to
+                });
+        });
+
+        const brandResult = await options
+            .transacting('settings')
+            .where('key', 'brand')
+            .select('value');
+
+        const brand = JSON.parse(brandResult[0].value);
+
+        logging.info(`Updating brand.primaryColor in settings to accent_color with value ${brand.primaryColor}`);
+
+        return await options
+            .transacting('settings')
+            .where('key', 'brand')
+            .update('key', 'accent_color')
+            .update('value', brand.primaryColor);
+    },
+
+    async down(options) {
+        await Promise.map(renameMapping, async (renameMap) => {
+            logging.info(`Renaming ${renameMap.to} to ${renameMap.from}`);
+
+            return await options
+                .transacting('settings')
+                .where('key', renameMap.to)
+                .update({
+                    key: renameMap.from
+                });
+        });
+
+        let accentColor = await options
+            .transacting('settings')
+            .where('key', 'accent_color')
+            .select('value');
+
+        const brand = accentColor[0].value;
+
+        logging.info(`Updating accent_color in settings to brand.primaryColor with value ${brand.primaryColor}`);
+
+        return await options
+            .transacting('settings')
+            .where('key', 'accent_color')
+            .update('key', 'brand')
+            .update('value', JSON.stringify({
+                brand: {
+                    primaryColor: ''
```

Comment:
Should this not be `brand.primaryColor`?

Reply:
@naz +1 the `down` is not restoring the color, also the value should be used directly as `brand` since its been set [here](https://github.com/TryGhost/Ghost/pull/11946/files#diff-3cff13c24fe1ebdc74b6c3630f01eceaR67)

Reply:
hm. valid point. having a look :+1:

Reply:
Have pushed out a fix, but it's not visible on the closed PR. Will open up a new one for clear comparison. Branch naming didn't change fyi, so you could just pull and test again